src/mixins/user-info.js

```diff
@@ -0,0 +1,38 @@
+import { mapGetters } from 'vuex';
+
+import { getIdenticonAvatar } from '~/util/api';
+
+export const createUserInfoMixin = ({ propKey = 'User' } = {}) => {
+  const getPropName = (label, prefix = '') => {
```

Comment:
How about just take a template string as param then call .replace? e.g. getPropName('Is{prop}CivicLiker')